MunchkinLevelCounter/UI/Charts/ChartsPresenter.swift

```diff
@@ -26,28 +27,29 @@ class ChartsPresenter: Presenter {
     
     func loadChartData(type: ScoreType) {
         loggingPrint("Loading chart data")
-        disposable = dataManager.getLineData(type: type.rawValue)
+        dataManager.getLineData(type: type.rawValue)
             .subscribeOn(ConcurrentDispatchQueueScheduler(qos: DispatchQoS.background))
             .observeOn(MainScheduler.instance)
             .subscribe(onNext: { lineChartData in
                     self.chartsView?.showPlayersCharts(chartData: lineChartData)
                 }
             )
+            .disposed(by: disposeBag)
     }
     
     func loadPlayers(sortType: ScoreType) {
-        disposable = dataManager.getPlayers(sortType: sortType)
+        dataManager.getPlayers(sortType: sortType)
             .subscribeOn(ConcurrentDispatchQueueScheduler(qos: DispatchQoS.background))
             .observeOn(MainScheduler.instance)
             .subscribe(onNext: { players in
                     self.chartsView?.showPlayersList(players: players)
                 }
             )
+            .disposed(by: disposeBag)
     }
     
     func detachView() {
         chartsView = nil
-        disposable?.dispose()
```

Comment:
> Every subscribe creates a retain cycle inside Rx’s logic. Thanks to it you don’t have to keep a strong reference to the button.rx.tap observable in above example. However, you have to break the retain cycle at some point. To do so, you have to call dispose() on Disposable which is an output from subscribe. Usually, you have more than a one Rx subscription inside UIViewController and to make disposing easier, you can add any Disposable into a DisposeBag. It’s just an array of Disposables which, on dealloc, goes through all disposables and dispose them. Disposables are disposed when the DisposeBag is deallocated which deallocs when its owner is deallocated. 

But here it about DisposeBag inside ViewController rather than in Presenter. So at this point I not perfectly sure is it will behaive same way.